impl/sql/mutations/mutations.go

```diff
@@ -35,7 +35,7 @@ const (
 	VALUES (?, ?, ?);`
 	readRangeExpr = `
   	SELECT Sequence, Mutation FROM Mutations
-  	WHERE MapID = ? AND Sequence > ?
+  	WHERE MapID = ? AND Sequence > ? AND Sequence <= ?
```

Comment:
I don't think so. There's no way to know the exact starting sequence giving that we only store the highest sequence number in each SignedMapRoot. I remember we decided that we want a range of ( ] to cover implementations using time-based sequence numbers.